modules/otel-config-instrumentation.adoc

```diff
@@ -252,6 +313,14 @@ instrumentation.opentelemetry.io/inject-java: "true"
 [id="otel-configuration-of-nodejs-auto-instrumentation_{context}"]
 == Configuration of the Node.js auto-instrumentation
 
+:FeatureName: The Node.js auto-instrumentation
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+By default, this feature injects unsupported, upstream instrumentation libraries.
```

Comment:
```suggestion
By default, this feature injects unsupported upstream instrumentation libraries.
```

I don't think comma is required here. Hence I suggest to remove that.

Reply:
Thank you for the suggestion. Using a comma here is correct in my understanding of punctuation.